packages/mux-uploader-react/src/useCombinedRefs.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useRef } from 'react';
```

Comment:
thought(non-blocking): We should consider eventually moving this into its own "shared" package (or put all of our react-relevant reusable code into a single shared package) to avoid the redundant code.